docs/pages/billing/usage-based-pricing.mdx

```diff
@@ -80,8 +80,8 @@ The global edge bandwidth calculation is:
 | Bandwidth used to send updates                    | 20 updates \* 5 MiB \* 10,000 users | 976.5625 GiB |
 |                                                   |
 | Bandwidth included in plan                        |                                     | 100 GiB      |
-| Bandwidth included with 9,000 extra updated users | 9,000 \* 40 MiB                     | 351.5625 GiB |
-| **Total**                                         | **976.5625 - 100 - 351.5625**       | **525 GiB**  |
+| Bandwidth included with 7,000 extra updated users | 7,000 \* 40 MiB                     | 273.4375 GiB |
```

Comment:
Can you please run `yarn run lint --fix` to apply Prettier formatting to this table? It'll help resolving the warnings below.